gulpfile.js/tasks/lint.js

```diff
@@ -0,0 +1,36 @@
+const gulp = require('gulp');
+const gulpUtil = require('gulp-util');
+const minimist = require('minimist');
+const spawn = require('child_process').spawn;
+const stylelint = require('stylelint');
+
+/**
+ * Lints the CSS for errors.
+ */
+function _lintCss() {
+  // Pass all command line flags to EsLint.
+  const args = minimist( process.argv.slice( 2 ) );
+  const willFix = args.fix;
+  let willFixFlag = '';
+
+  if (willFix) {
+    willFixFlag = '--fix';
+  }
+
+  return new Promise( ( resolve, reject ) => {
+    spawn(
+      'node_modules/stylelint/bin/stylelint.js',
+      ['client/**/*.scss', 'wagtail/**/static_src/**/scss/**/*.scss', willFixFlag],
+      { stdio: 'inherit' }
+    ).once( 'close', code => {
+      if ( code ) {
+        gulpUtil.log( 'Stylelint exited with code ' + code );
+        reject();
+      }
+      gulpUtil.log( 'Stylelint done!' );
+      resolve();
```

Comment:
It's so you can pass arguments to stylelint, specifically the `--fix` flag to autofix errors. Perhaps optional arguments can be passed through to the command in package.json, but I wasn't sure how that's done. Since you have an existing gulp task infrastructure I went with this. I'd recommend supporting the same flag for ESLint.